test/integration/hooks/hooks.test.js

```diff
@@ -134,6 +134,34 @@ describe(Support.getTestDialectTeaser('Hooks'), () => {
       });
     });
 
+    describe('After acquire connection hook', () => {
```

Comment:
Add unit tests for this change instead https://github.com/sequelize/sequelize/blob/master/test/unit/connection-manager.test.js